plugins/GSdx/GSTextureCache.cpp

```diff
@@ -199,8 +199,9 @@ GSTextureCache::Source* GSTextureCache::LookupSource(const GIFRegTEX0& TEX0, con
 
 	list<Source*>& m = m_src.m_map[TEX0.TBP0 >> 5];
 
-
-	for(list<Source*>::iterator i = m.begin(); i != m.end(); ++i)
+	list<Source*>::iterator m_begin = m.begin();
```

Comment:
``m_`` is generally used to name class members, maybe not a good name for a local variable?